<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWorkTool</class>
 <widget class="QWidget" name="HomeWorkTool">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home Work Tool</string>
  </property>
  <widget class="QLabel" name="lHomeWork">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Home Work Tool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBClose">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Create New One</string>
   </property>
   <widget class="QPushButton" name="pBNewHomeWork">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBNewHomeWrok">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home Work</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBNewShotNote">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>46</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Short Note</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Open Existing one</string>
   </property>
   <widget class="QPushButton" name="pBOpenExistingHw">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBOpenHomeWork">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>46</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home Work</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBOpenNote">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>46</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Short Note</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>HomeWorkTool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
